### DISCUSSION TITLE
Accessibility improvements in Buttons

### DISCUSSION THREAD

**Discussion by Ro-Matos** [Telefonica/mistica-design#1435] [Proposals]
### Goal

- 1 - Add a optional prop `aria-label` to Button component. Sometimes the semantic label is tied to the context that there isn't enough info for screen readers, should be possible to allow including an aria-label that provides additional context.

Example: Visible label is "See more" and aria-label is "See more products".

- 2 - Add prop `aria-disabled` to buttons in 'disabled' state. The aria-disabled state indicates that the element is perceivable but disabled. Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

Some screen-readers jump the button when only the prop `disabled` is use.

### Context

In our accessibility specs of flows, developed by the design teams of APP Vivo, this cases have frequently documented. But the developers are not able to assign the properties in the code.

### Impact

Improvements and compliance with accessibility guidelines.

### Discussion Questions

None

**Comment by yceballost** [comment]
Thanks Rodrigo! We are going to include this improvement in buttons. Would you like to include this information in buttons specs?

**Reply by Ro-Matos** [reply]
Great @yceballost! 
Yes, I can include this in specs. You can create a branch in button specs to include a a11y section?

**Reply by yceballost** [reply]
sure!
https://www.figma.com/file/8QZS0a9TOk6Cjpprcb4x0A/%231443---Accessibility-improvements-in-Buttons?type=design&node-id=0-1&mode=design

**Reply by Ro-Matos** [reply]
Done!
Note: I inserted in the documentation only the points referring to the proposal.
Thanks @yceballost!
